spec/index.html

```diff
@@ -1152,6 +1152,23 @@ <h3>The XML Schema Built-in Datatypes</h3>
         model.</li>
     </ul>
 
+    <p class="note">The <a>value spaces</a>
+      of <a data-cite="xmlschema11-2#double"><code>xsd:double</code></a> and
+      <a data-cite="xmlschema11-2#float"><code>xsd:float</code></a> do not
+      include all decimal numbers. For every literal of
+      any of these two datatypes, the value given as the <a>lexical form</a>
+      of the literal is rounded to a value that can be represented as an
+      <a href="http://ieeexplore.ieee.org/servlet/opac?punumber=4610933">IEEE 754-2008</a>
+      binary floating point representation of the corresponding precision.
+      As a consequence, not all decimal numbers are accurately captured by
+      using these two datatypes. For instance, the value of the literal with
```

Comment:
The sentence is redundant to the first sentence and the next sentence. To keep the note short, it could be skipped.

```suggestion
      
      For instance, the value of the literal with
```